tests/test_feature_extraction.py

```diff
@@ -65,3 +73,811 @@ def test_logmel_length():
         assert torch.all(mel_center.size()[1] == length_center)
         mel_no_center, length_no_center = fe_no_center(audio, audio_length)
         assert torch.all(mel_no_center.size()[1] == length_no_center)
+
+
+def test_rasr_compatible():
+    try:
+        from i6_core.lib.rasr_cache import FileArchive
+    except ImportError:
+        raise unittest.SkipTest("i6_core not available")
+    try:
+        import soundfile
+    except ImportError:
+        raise unittest.SkipTest("soundfile not available")
+    rasr_feature_extractor_bin_path = (
+        "/work/tools22/asr/rasr/rasr_onnx_haswell_0623/arch/linux-x86_64-standard/"
+        "feature-extraction.linux-x86_64-standard"
+    )
```

Comment:
As these test depend on the i6_cluster, should we remove them before finally merging?

Reply:
I would leave them for reference. They will just raise `SkipTest` when it does not find the library, so not cause the tests to break elsewhere.